docs/data/rpc/api-reference/methods/README.mdx

```diff
@@ -8,3 +8,20 @@ import DocCardList from "@theme/DocCardList";
 All you need to know about available RPC methods, parameters and responses, and making RPC requests.
 
 <DocCardList />
+
+Don't know which endpoint you need to get what you want? A lot of the returned fields are deeply-nested [XDR structures](/docs/learn/encyclopedia/data-format/xdr.mdx#more-about-xdr), so it can be hard to figure out what kind of information is available in each of these. Here's a bit of a dive into what the "workhorse" endpoints provide, in decreasing order of granularity:
+
+- [`getLedgers`](./getLedgers.mdx) operates at the block level, providing you with the full, complete details of what occurred during application of that ledger (known as "ledger metadata", defined in the protocol by the [`LedgerCloseMetaV1`](https://github.com/stellar/stellar-xdr/blob/v22.0/Stellar-ledger.x#L508-L537) structure). Each of the subsequent endpoints is just a microscope into a subset of the data available provided by this endpoint. Metadata includes things like:
+  - Details for recreating the blockchain's state (see [Ledger Headers](/docs/learn/encyclopedia/network-configuration/ledger-headers.mdx) for more).
+  - The consensus information that led to the block closing (see [Stellar Consensus Protocol](/docs/learn/fundamentals/stellar-consensus-protocol.mdx)).
+  - The set of transactions, their respective operations, and the results of applying those transactions in this block (see [Transactions](/docs/learn/fundamentals/transactions/operations-and-transactions)).
+- [`getTransaction(s)`](./getTransactions.mdx) operates across a span of ledgers or on a single transaction hash depending on the variant. The structured data here includes details such as:
+  - The exact transaction structure ("envelope") that was submitted.
+  - Results for each of the operations within the transaction.
+  - All side-effects to ledger state that occurred as a result of this transaction.
+- `getEvents` lets you search for events that occurred over a ledger range. Each event is made up of topics (which you can filter on) and data which are `ScVal`s, Stellar's generic "value" type (see [Contract Development](/docs/learn/encyclopedia/contract-development/types/built-in-types) for more).
```

Comment:
might be worth it to give a brief description of what events are (E.g. events are emitted by smart contracts for the purpose of communicating meaningful state changes to the smart contract to off chain indexers).

Reply:
Nice! Incorporated this and added a cross-reference to https://developers-pr1369.previews.kube001.services.stellar-ops.com/docs/learn/encyclopedia/contract-development/events, too.